deepinv/models/icnn.py

```diff
@@ -4,6 +4,139 @@
 from torch import nn
 
 
+class tiny_ICNN(nn.Module):
+    r"""
+    Small Input Convex Neural Network.
+
```

Comment:
can you add a small explanation of what is this architecture (or a ref to a paper?)